tests/integration/compact_test.py

```diff
@@ -0,0 +1,141 @@
+import typing
+
+from mock.mock import patch
+
+import hazelcast
+from hazelcast.compact import CompactSchemaService
+from hazelcast.errors import IllegalStateError
+from hazelcast.future import ImmediateFuture, ImmediateExceptionFuture
+from hazelcast.serialization.api import CompactSerializer, CompactWriter, CompactReader
+from tests.base import HazelcastTestCase
+from tests.util import random_string
+
+
+# The tests under this file are meant to be not
+# run with the backward compatibility tests, as
+# they test implementation details through private
+# APIs. If you are going to add a test to this file,
+# make sure that it fits the description above. If not,
+# add it to tests under the backward_compatible folder.
+
+
+class CompactSchemaReplicationRetryTest(HazelcastTestCase):
+    rc = None
+    cluster = None
+    retry_count = None
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.rc = cls.create_rc()
+        cls.cluster = cls.create_cluster(cls.rc, None)
+        cls.cluster.start_member()
+        cls.cluster.start_member()
+        cls.retry_count = CompactSchemaService._SEND_SCHEMA_RETRY_COUNT
```

Comment:
Renaming `retry_count` to `backup_retry_count` may makes its purpose clearer.

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/615/commits/4271ad0158e8045df1c5ebbf7aff792b72fd5e37